system/netdata.service.in

```diff
@@ -12,7 +12,9 @@ Group=netdata
 RuntimeDirectory=netdata
 RuntimeDirectoryMode=0775
 PIDFile=@localstatedir_POST@/run/netdata/netdata.pid
-ExecStart=@sbindir_POST@/netdata -P @localstatedir_POST@/run/netdata/netdata.pid -D -W set global 'process scheduling policy' 'keep' -W set global 'OOM score' 'keep'
+Environment=DAEMONOPTS="-P @localstatedir_POST@/run/netdata/netdata.pid"
+EnvironmentFile=-/etc/sysconfig/netdata
```

Comment:
Well, that's because i'm used to work on RHEL based distros, and they use "/etc/sysconfig" as standard directory for daemon configuration files.
In Debian based distros i believe they use the "/etc/default" directory.
Other solution would be to use a @sbindir_POST@ (or any other install variable), so that it always uses the same directory, independent from system where it's installed.
But I'd prefer to use the standard directory for each system, do that it's easier to find.

Reply:
How about using @configdir_POST@ instead ?

Reply:
Better than @sbindir_POST@. 
I am afraid that I do not have all the time I would like to make a better request, which would require less revision.
Thanks for your work.

Reply:
i finally connected and changed it (i've been out last days)